pages/_slug/index.vue

```diff
@@ -18,7 +18,11 @@ export default {
 	},
 	head () {
 		return {
-			titleTemplate: this.title + " | %s"
+			titleTemplate: this.title + " | %s",
+			meta: [
+				{ hid: "og:title", name: "og:title", content: this.title + " | We Count" },
+				{ hid: "og:url", name: "og:url", content: Config.appBaseUrl + this.$nuxt.$route.fullPath }
```

Comment:
IT seems the `og:url` is not being assigned to the `property` attribute.  The same for `og:title` and `og:image`. See https://ogp.me/

Reply:
Resolved in 848dced.